### PR TITLE
fix usage and quoting to fix hub

### PR DESCRIPTION
**Pull request by coapacetic**
This is a:
- [X] bug fix PR with no breaking changes — please ensure the base branch is `master`
- [ ] new functionality — please ensure the base branch is the latest `dev/` branch
- [ ] a breaking change — please ensure the base branch is the latest `dev/` branch

## Description & motivation
hub.getdbt.com isn't looking so great.  I think the culprit is the use of a `>` for quoting in markdown.  I'm submitting this PR to hopefully then see this fixed over at hub.getdbt.com.  While I was here, I also ensured that all `usage` headings were bolder for easier reading.

Here is an screenshot of hub.getdbt.com that isn't looking so hot right now.

image

## Checklist
- [X] I have verified that these changes work locally on the following warehouses (Note: it's okay if you do not have access to all warehouses, this helps us understand what has been covered)
    - [ ] BigQuery
    - [ ] Postgres
    - [ ] Redshift
    - [ ] Snowflake
- [X] I have updated the README.md (if applicable)
- [X] I have added tests & descriptions to my models (and macros if applicable)
- [X] I have added an entry to CHANGELOG.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -508,10 +513,13 @@ Generate drop statements for all Relations that match a naming pattern:
 
 #### get_relations_by_prefix ([source](macros/sql/get_relations_by_prefix.sql))
 > This macro will soon be deprecated in favor of the more flexible `get_relations_by_pattern` macro (above)
+
 Returns a list of [Relations](https://docs.getdbt.com/docs/writing-code-in-dbt/class-reference/#relation)
 that match a given prefix, with an optional exclusion pattern. It's particularly
```
Comment (coapacetic) [inline README.md:518]: I think this little bit of whitespace should do the trick 😄

**Comment by coapacetic** [comment]
(closing out all things that might be open on my list) feel free to merge, not merge, close / whatever seems appropriate here 😄

**Comment by clrcrl** [comment]
Changing the base to `dev/0.7.0` to avoid making another `0.6.x` release

**Review by clrcrl — APPROVED** [review]
thanks Coap!

**Pull request merged**